### PR TITLE
CI update - .NET 8 RC 2

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: xml/Microsoft.Extensions.Compliance.Redaction/XXHash3RedactorOptions.xml
Comment (gewarren) [inline xml/Microsoft.Extensions.Compliance.Redaction/XXHash3RedactorOptions.xml:1]: @huangmin-ms Probably the casing of the file name should have changed too. Should I log a bug?

**Review by carlossanlop — APPROVED** [review]

File: xml/Microsoft.Extensions.Compliance.Redaction/XXHash3RedactorOptions.xml
Comment (huangmin-ms) [reply xml/Microsoft.Extensions.Compliance.Redaction/XXHash3RedactorOptions.xml:1]: Windows is case insensitive system. This should be expected.

Comment (gewarren) [reply xml/Microsoft.Extensions.Compliance.Redaction/XXHash3RedactorOptions.xml:1]: @huangmin-ms Yes, but it no longer matches the name of the type. I believe if you use `git mv` then you can rename a file that only has a case change. Should I rename it manually?

Comment (gewarren) [reply xml/Microsoft.Extensions.Compliance.Redaction/XXHash3RedactorOptions.xml:1]: I used `git mv` to rename it in this PR: https://github.com/dotnet/dotnet-api-docs/pull/9387

**Pull request merged**